src/stories/CookieConsent.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
I think it's also worth adding `src/stories` path to `exclude` in `tsconfig.json`

Reply:
It is done!